src/index.js

```diff
@@ -206,6 +209,15 @@ class Cropper extends React.Component {
     const point = Cropper.getMousePoint(e)
     const newZoom = this.props.zoom - (e.deltaY * this.props.zoomSpeed) / 200
     this.setNewZoom(newZoom, point)
+
+    !this.state.onWheelStart &&
+      this.setState({ onWheelStart: true }, () => this.props.onInteractionStart())
+
+    clearTimeout(this.wheelTimer)
+    this.wheelTimer = setTimeout(
+      () => this.setState({ onWheelStart: false }, () => this.props.onInteractionEnd()),
+      250
```

Comment:
Is this enough? I still have to play with it to forge an opinion, I'm not on the computer now ^^